org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/XMLDocumentSymbolsTest.java

```diff
@@ -27,12 +28,14 @@
  */
 public class XMLDocumentSymbolsTest {
 
+	private static final int defaultSymbolLimit = 5000;

```

Comment:
Remove this default value and add a new signature to XMLAssert (without symbol limit) with limit=0